framework/source/class/qx/test/theme/simple/Appearance.js

```diff
@@ -0,0 +1,159 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2004-2016 1&1 Internet AG, Germany, http://www.1und1.de
+
+   License:
+     LGPL: http://www.gnu.org/licenses/lgpl.html
+     EPL: http://www.eclipse.org/org/documents/epl-v10.php
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+     * William Opandi (woprandi)
+
+************************************************************************ */
+qx.Class.define("qx.test.theme.simple.Appearance",
+{
+  extend : qx.test.unit.TestCase,
+
```

Comment:
The `generate test-source` only showed warnings, no errors, so travis did not stop.